tests/testthat/test-proportional_hazards-glmnet.R

```diff
@@ -164,3 +164,57 @@ test_that("survival probabilities - non-stratified model", {
   )
 
 })
+
+
+test_that("survival probabilities - stratified model", {
+
+  cox_spec <- proportional_hazards(penalty = 0.123) %>%
+    set_mode("censored regression") %>%
+    set_engine("glmnet")
+
+  set.seed(14)
+  expect_error(
```

Comment:
Can you also add test cases with induced missing values?

Reply:
to be addressed with #64